experiment/seccomp/src/instruction/consts.rs

```diff
@@ -75,8 +97,25 @@ pub const fn seccomp_data_arg_size() -> u8 {
     8
 }
 
-pub const fn seccomp_data_args_offset() -> u8 {
-    offset_of!(SeccompData, args) as u8
+/// Returns the offset of the `args` field in `SeccompData` for the given index.
+/// 
+/// # Arguments
+/// 
+/// * `index` - The index of the argument (0–5).
+/// 
+/// # Returns
+/// 
+/// A `Result` containing the offset as `u8` if the index is valid, or an error message if the index is out of range.
+pub const fn seccomp_data_args_offset(index: u8) -> Result<u8, &'static str> {
+    match index {
+        0 => Ok(offset_of!(SeccompData, args) as u8),
+        1 => Ok(offset_of!(SeccompData, args) as u8 + 8),
+        2 => Ok(offset_of!(SeccompData, args) as u8 + 16),
+        3 => Ok(offset_of!(SeccompData, args) as u8 + 24),
+        4 => Ok(offset_of!(SeccompData, args) as u8 + 32),
+        5 => Ok(offset_of!(SeccompData, args) as u8 + 48),
+        _ => Err("Index out of range. Valid indices are 0–5."),
```

Comment:
指数6番目のオフセットは40が正しいと思うんだけど、どうかな？

もしそうであれば、レンジmatchを使うといいと思う。

```suggestion
        [0..=5] => Ok(offset_of!(SeccompData, args) as u8) + (index * 8)),
        _ => Err("Index out of range. Valid indices are 0–5."),
```

Reply:
@ianlewis 

レビューありがとうございます。返信が遅くなり大変失礼しました。
コメントの通りここは6つ目のオフセットは40が正しいので修正しました。

https://github.com/utam0k/youki/pull/541/commits/eeb8a4cfeaaa83fb0f9d22ab17f91a910fac9759